sections/debt/components/DebtHedgingInfoPanel/DebtHedgingChart.tsx

```diff
@@ -45,7 +45,9 @@ const DebtHedgingChart: React.FC = () => {
 						interval="preserveEnd"
 						tick={{ fontSize: 10, fill: colors.white, fontFamily: fonts.mono }}
 						tickLine={false}
-						tickFormatter={(tick) => format(new Date(tick * 1000), 'd MMM yy').toUpperCase()}
+						tickFormatter={(tick) =>
+							format(new Date(tick * 1000 || new Date().getTime()), 'd MMM yy').toUpperCase()
+						}
```

Comment:
Did we have cases where the tick produced an invalid Date? Can we apply a filter on the data to remove those data points rather than using an incorrect timestamp?

Reply:
if you look at prod. The x axis starts at 1970.1.1. I console logged the tick and it is undefined. Dunno what causes it. That is why I went with that solution

Reply:
But if you think a filter function is helpful here, I can write one

Reply:
Hmm yea I guess we dont want to filter out data without a timestamp. 
Took a quick look, data coming from here:
https://github.com/Synthetixio/staking/blob/dev/hooks/useGlobalHistoricalDebtData.ts#L49
We could just not push if timestamp is missing

Reply:
if you look at line 24 and 28 in that file, you see we don't query for timestamps. Cause the endpoint don't provide timestamps. I removed the prop entirely.
I'm also thinking, we should show maybe some graph from the uniswap pool, when it exists? Would make more sense? But not sure for now.